Azure/sonic-buildimage
# [201911][Mellanox] update FW to *.2008.2314

Signed-off-by: Stepan Blyschak <stepanb@nvidia.com>



#### Why I did it

Bring in a fix for thermal shutdown observed while executing warm-reboot:
- All | prevent FW access during ISSU

#### How I did it
Updated submodule and updated relevant versions in makefile.

#### How to verify it
Build and run on the switch.
Perform 50 iterations of warm-reboot and observe no thermal shutdown.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
